src/pages/Dictionary/Dictionary.jsx

```diff
@@ -30,16 +46,41 @@ const Dictionary = () => {
   } else if (status === 'success') {
     return (
       <div className="Terms">
-        {entries.map((entry, index) => (
-          <div key={index}>
-            <h2>{terms[entry.termid]}</h2>
-            <h3>Definition</h3>
-            <p>{entry.definition}</p>
-            <h3>Example</h3>
-            <p>{entry.example}</p>
-            <br />
-          </div>
-        ))}
+        <div className="px-8 py-8">
+          {entries.map((entry, index) => (
+            <div key={index} className="mb-4">
+              <div className="bg-blue-900 p-5 border-4 border-gray-200">
+                <Text h1 className="term">
+                  {terms[entry.termid]}
+                </Text>
+                <Text h2>Definition</Text>
+                <Text h3 className="definition">
+                  {entry.definition}
+                </Text>
+                <Text h2>Example</Text>
+                <Text h3 className="example">
```

Comment:
you don't need to create a separate className for this and call the className in CSS file because we are using Tailwind..